apps/backend/.env.docker

```diff
@@ -0,0 +1,33 @@
+STACK_BASE_URL=http://localhost:8102
+STACK_SERVER_SECRET=23-wuNpik0gIW4mruTz25rbIvhuuvZFrLOLtL7J4tyo
+
+# Postgres connection strings
+STACK_DATABASE_CONNECTION_STRING=postgres://stack:stack123@postgres:5432/stack
+STACK_DIRECT_DATABASE_CONNECTION_STRING=postgres://stack:stack123@postgres:5432/stack
+
+# optionally skip migrations on container startup
+STACK_SKIP_MIGRATIONS="false"
+
+# run Prisma seed script on container startup
+STACK_RUN_SEED_SCRIPT="true"
+
+# Self-host seed script options
+# Optionally create a default admin user and disable signups to the platform when the seed script is run.
+# This can be useful for self hosted deployments where you want to restrict access to the Stack "internal" project.
+# If not specified, no user will be created and you will have to create the first user manually by signing up.
+# STACK_DEFAULT_DASHBOARD_USER_EMAIL=admin@example.com
+# STACK_DEFAULT_DASHBOARD_USER_PASSWORD=admin123
+# STACK_DEFAULT_DASHBOARD_USER_DISPLAY_NAME=Admin
+
+# Sign up
+STACK_SIGN_UP_DISABLED= # set to "true" to disable sign up on the platform ("internal" project). If set to "true", the default admin user above will need to be created to log in
+
+STACK_EMAIL_HOST=inbucket
+STACK_EMAIL_PORT=2500
+STACK_EMAIL_SECURE=false
+STACK_EMAIL_USERNAME=does not matter, ignored by Inbucket
+STACK_EMAIL_PASSWORD=does not matter, ignored by Inbucket
+STACK_EMAIL_SENDER=noreply@example.com
+
+STACK_SVIX_SERVER_URL=http://localhost:8113
+STACK_SVIX_API_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE2NTUxNDA2MzksImV4cCI6MTk3MDUwMDYzOSwibmJmIjoxNjU1MTQwNjM5LCJpc3MiOiJzdml4LXNlcnZlciIsInN1YiI6Im9yZ18yM3JiOFlkR3FNVDBxSXpwZ0d3ZFhmSGlyTXUifQ.En8w77ZJWbd0qrMlHHupHUB-4cx17RfzFykseg95SUk
```

Comment:
Is this API key created by default? Does it not expire? (seems to be a JWT)

Reply:
This is just for the local svix. Also this .env file should actually be removed, forgot to do that.

Reply:
FYI, you can call it .env.docker.local or .env.docker.untracked so it won't be committed to git